File: 18_lombok_Tomassetti/src/main/java/com/tschlatd/_lombok_wirtschaftstag/models/Company.java
package com.scarc._lombok_tomassetti.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Company {
    private String name;
    private String zipTown;
    private String street;
    private String phone;
    private String eMail;
    private String replyTo;
    private String comments;

    private List<Participation> participatesIn;
}

File: KOTLIN/Wirtschaftstag/Kotlin_Wirtschaftstag/androidApp/src/main/java/me/scarc/androidApp/data/Company.kt
package me.scarc.androidApp

import me.scarc.androidApp.data.Participation

data class Company(
    var id: Int,
    var name: String,
    var zipTown: String,
    var street: String,
    var phone: String,
    var replyTo: String,
    var comments: String,
    var participatesIn: List<Participation>,
    var email: String,
) {
    override fun toString(): String {
        return "Company #$id\n$name in $zipTown, $street.\nContact: $replyTo\nEmail: $email\nPhone: $phone \n" +
                "Comments: $comments"
    }
}